### PR TITLE
New PGCE content

### PR DESCRIPTION
**Pull request by EllieNodder**
### Trello card

https://trello.com/c/C2qkWnIS/3342-expand-pgce-page

### Context

Our PGCE content is quite thin and doesn't answer a lot of questions users have around PGCEs. Updating the content with more info will be better for users and also hopefully boost our SEO by capturing more relevant keywords.

### CODE REVIEW

File: app/views/content/what-is-a-pgce.md [hunk 1]
```diff
@@ -19,28 +26,64 @@ keywords:
...
+Many teacher training courses also offer you a postgraduate qualification such as a PGCE, but you do not need one to teach.
 
-You can do a PGCE course at a university, or through a school-led teacher training programme.
+## Teacher training courses
 
-You can do a PGCE in primary education, secondary education, or further/adult education.
+You can do a teacher training course that leads to:
 
-## PGDE
+* just QTS
```
[the remark this answers is not in the document]
Comment (gemmadallmandfe) [reply app/views/content/what-is-a-pgce.md:45]: If so, the paragraphs below would also need changing

File: app/views/content/what-is-a-pgce.md [hunk 2]
```diff
@@ -19,28 +26,64 @@ keywords:
...
+* QTS with PGCE
 
-A postgraduate diploma in education (PGDE) course is very similar to a PGCE, the difference being the amount of academic credits you are awarded at the end of your course.
+Getting just QTS means that you’re qualified to teach in most schools in England. It may also mean that your course fees are lower, you submit fewer assignments, and the course could be shorter.
 
-## Teaching qualifications you need
+Getting just a PGCE means you will not be qualified to teach in most primary, secondary and special schools in England.
 
-If you want to be a primary school teacher or secondary school teacher in a state school in England, you will need to get qualified teacher status (QTS). You don’t need a PGCE or PGDE to become a teacher, but many courses offer it alongside QTS.
+[Read about Nathan who chose to get QTS with PGCE](/blog/salaried-teacher-training-classroom-learning).
```
Comment (gemmadallmandfe) [inline app/views/content/what-is-a-pgce.md:53]: Do we have a more recent blog post that would be relevant? Now that we have the banner at the top of the page re NQTs, the blog post feels noticeably old

File: app/views/content/what-is-a-pgce.md [hunk 3]
```diff
@@ -19,28 +26,64 @@ keywords:
...
 
-<a href="/what-is-qts" class="button">Find out more about qualified teacher status (QTS)</a>
+## Benefits of having a PGCE
+
+A PGCE can give you a better understanding of teaching practices, educational research and theory.
+
+It can also be beneficial if you want to teach in other countries because it’s an internationally recognised academic qualification. However, you should always check the local requirements of the country you’re looking to teach in.
+
+It can also award up to 60 credits at Master’s level which you could use towards a full Master’s qualification. This is usually 180 credits.
```
Comment (gemmadallmandfe) [inline app/views/content/what-is-a-pgce.md:61]: I'm presuming this would be in the future and not linked to ITT? Do we need to make that clear?

**Review by gemmadallmandfe — COMMENTED** [review]
Looks good @EllieNodder - a few minor suggestions but happy for you to ignore if needs be!

File: app/views/content/what-is-a-pgce.md (hunk 2 above)
Comment (EllieNodder) [reply app/views/content/what-is-a-pgce.md:53]: There is this [PGCE blog post[(https://getintoteaching.education.gov.uk/blog/what-to-expect-on-your-teacher-training) currently in the related content section. The only thing is, I don't think it's very clear whether it's talking about a PGCE only course, or a QTS with PGCE course.

**Review by ethax-ross — APPROVED** [review]
Looks good; its a bit confusing with it not being in the train to be a teacher section but pulling in promos from there, but we can maybe refactor the promos to make it clearer they're used outside of the train to be a teacher content at some point in the future

**Pull request merged**